README.md

```diff
@@ -605,6 +615,53 @@ The provenance for the `title` and `nightly` property values. Can be one of:
 The `source` property is always set.
 
 
+## Spec identifiers
+
+An entry in browser-specs contains properties that can be viewed as
+identifiers: [`shortname`](#shortname), [`url`](#url), and
+[`nightly.url`](#nightlyurl). Please note that these identifiers are not fully
+stable.
+
+The `shortname` property should remain mostly stable over time. The `shortname`
+may still change though, for instance when a W3C specification starts being
+published as a TR document with a shortname that is different from the one used
+during incubation, or when an IETF specification gets published as an RFC.
+Starting in July 2023, when the `shortname` of a specification changes in
+browser-specs, the previous `shortname` gets added to a
+[`formerNames`](#formernames) property. This makes it possible to track a
+specification entry over time in browser-specs.
+
+The `url` property contains a URL of the specification that can be regarded as
+canonical and mostly stable too, but that URL will typically change when a
+specification starts getting published as a formal technical document, or when
+a specification transitions from one organization or group to another one.
+
+The `nightly.url` property is the least stable identifier of a specification.
+That URL may be under the control of an individual or group, who may decide to
+change the URL at any time. Or it may be affected by a change of status. For
+instance, the `nightly.url` property will change when a W3C spec incubated in
+the Web Platform Incubator Community Group (WICG) transitions to a Working
+Group, or when a new version of an IETF draft gets published.
+
+If your project tracks specifications over time and relies on browser-specs to
+gather information about these specifications, you will need to record the
+`shortname` of the specifications you're tracking, and apply the following
+algorithm to find the relevant specification entry in browser-specs:
+
+1. Look for an entry in browser-specs whose `shortname` matches the recorded
+shortname. If one is found, that is the relevant specification entry.
+2. Look for the first entry in browser-specs that has the recorded shortname in
```

Comment:
should we add a test that no two specs should overlapping formerNames? or is that not something we guarantee either?

Reply:
I hesitated but that seems something that we can at least guarantee for the foreseeable future. I'm adding a test and updating the algorithm to note the guarantee.